clusterloader2/testing/list/config.yaml

```diff
@@ -0,0 +1,192 @@
+# List benchmark
+{{$namespaceNumber := DefaultParam .CL2_LIST_NAMESPACE_NUMBER 2}}
```

Comment:
can we just use `NODES_PER_NAMESPACE` variable that we use for other test configs ? Wondering if we really need this ? Could it be a factor of nodes and determine from that ?

Reply:
Updated